### ISSUE TITLE
Feature request: line numbers

### ISSUE THREAD

**Issue by lukyanov** [jakob/TableTool#29]
Thank you for the great and very lightweight product!

It would be really useful if the app had line numbers in the table. Sometimes you refer to a line in a csv file by its number.

**Comment by ajunior** [comment]
Maybe a total number of lines could be great in the status bar.

**Comment by fulldecent** [comment]
Please propose an UI mock up of how the status bar should be displayed

**Comment by redcapua** [comment]
To show row number just click on menu
Window -> Show line number
![show line number screen shot](https://user-images.githubusercontent.com/9165541/49611298-c71a9f00-f9a9-11e8-8966-2bd46bbb7f5e.png)

**Comment by maiksd** [comment]
The App Store version doesn't have that menu item.

**Comment by 2803media** [comment]
any update on this?

**Comment by deeTEEcee** [comment]
yeah, i think being aware of the line number is a pretty fundamental use case. (e.g: a way to check offsets for ordered data) would love it if it was added.

**Comment by vmred** [comment]
+++
